### PR TITLE
Tutorials

### PR DESCRIPTION
**Pull request by jslee02**
The purpose of this tutorial is to provide a quick introduction to using DART. We designed many hands-on exercises to make the learning effective and fun. To follow along with this tutorial, first locate the tutorial code in the directory: dart/tutorials. For each of the four tutorials, we provide the skeleton code as the starting point (e.g. tutorialMultiPendulum.cpp) and the final code as the answer to the tutorial (e.g. tutorialMultiPendulum-Finished.cpp). The examples are based on the APIs of DART 5.0.

[Here](http://dart.readthedocs.org/en/tutorials/) is the prerelease version of the tutorial site for this PR. Note that once this PR is merged the official tutorial site will be [this](http://dart.readthedocs.org/).

Any feedback is welcome!

### CODE REVIEW

**Comment by jslee02** [comment]
Waiting a day for further comments before merging.

File: apps/vehicle/Main.cpp [hunk 1]
```diff
@@ -52,7 +52,7 @@ int main(int argc, char* argv[])
   assert(myWorld != nullptr);
   Eigen::Vector3d gravity(0.0, -9.81, 0.0);
   myWorld->setGravity(gravity);
-
+
```
Comment (psigen) [inline apps/vehicle/Main.cpp]: Delete whitespace?

File: apps/vehicle/MyWindow.cpp [hunk 2]
```diff
@@ -51,8 +51,8 @@ void MyWindow::timeStepping() {
   dart::dynamics::SkeletonPtr vehicle = mWorld->getSkeleton("car_skeleton");
   assert(vehicle != 0);
 
-  size_t dof = vehicle->getNumDofs();
-
+  size_t dof = vehicle->getNumDofs();  
+
```
Comment (psigen) [inline apps/vehicle/MyWindow.cpp]: Delete whitespace?

**Comment by psigen** [comment]
Is this PR merging from some other bugfix branches?  It seems like there are some minor `const`-correctness and other small fixes in here.

**Comment by jslee02** [comment]
I don't think so. It seems those fixes were for the bugs found during the tutorial writing. Ideally, it would be good to create separate PR for the fixes but they are minor enough to be included in here I guess.

File: apps/vehicle/MyWindow.cpp (hunk 2 above)
Comment (jslee02) [inline apps/vehicle/MyWindow.cpp]: :+1:  Deleted

File: apps/vehicle/Main.cpp (hunk 1 above)
Comment (jslee02) [inline apps/vehicle/Main.cpp]: :+1:  Deleted

File: .gitignore [hunk 3]
```diff
@@ -23,3 +22,4 @@ TAGS
 *~
 /nbproject/
 /doxygen/html/
+docs/readthedocs/site
```
Comment (psigen) [inline .gitignore]: Newline at EOF?

**Comment by psigen** [comment]
These tutorials look really good!  Nice work guys.

File: .gitignore (hunk 3 above)
Comment (jslee02) [inline .gitignore]: :+1:  added

**Pull request merged**